src/System.Runtime.NumericsTestData/System.Runtime.NumericsTestData.csproj

```diff
@@ -0,0 +1 @@
+<Project Sdk="Microsoft.Build.NoTargets" />
```

Comment:
I copied this from other projects in this repo. 
1. When I do `dotnet build` in the root directory, what should the artifacts contain? 
2. Right now I only see a deps.json. Where is the text file that I added?

Reply:
The file is right but the name of the project is wrong. There should be a "." between "Numerics" and "TestData".

> When I do dotnet build in the root directory, what should the artifacts contain?

You aren't building anything in this project, you are only packaging. Hence try a `dotnet pack` instead.

Reply:
Ah of course, duh. Thanks, the package in the artifact does contain the txt file I added now.